src/Catty/ViewController/Download/Details/ProjectDetailStoreViewControllerExtension.swift

```diff
@@ -0,0 +1,411 @@
+        label?.layer.shadowColor = UIColor.white.cgColor
+        label?.layer.shadowOffset = CGSize(width: 0.0, height: 0.0)
+    }
+
+    func configureTextLabel(_ label: UILabel?, andHeight height: CGFloat) {
+        label?.backgroundColor = UIColor.clear
+        if Float(height) == kIpadScreenHeight {
+            label?.font = UIFont.boldSystemFont(ofSize: 18)
+        } else {
+            label?.font = UIFont.boldSystemFont(ofSize: 12)
+        }
+        label?.textColor = UIColor.textTint
+        label?.layer.shadowColor = UIColor.white.cgColor
+        label?.layer.shadowOffset = CGSize(width: 0.0, height: 0.0)
+    }
+
+    func setMaxHeightIfGreaterFor(_ view: UIView?, withHeight height: CGFloat) {
+        var frame = view?.frame
+        if (frame?.size.height ?? 0.0) < height {
+            frame?.size.height = height
+            view?.frame = frame ?? CGRect.zero
+        }
+    }
+
+    func configureDescriptionLabel(_ label: TTTAttributedLabel?) {
+        let height = self.height()
+        label?.lineBreakMode = NSLineBreakMode.byWordWrapping
+        label?.numberOfLines = 0
+        self.configureTextLabel(label, andHeight: height)
+        label?.enabledTextCheckingTypes = NSTextCheckingAllTypes
+        label?.verticalAlignment = TTTAttributedLabelVerticalAlignment.top
+
+        var mutableLinkAttributes: [AnyHashable: Any] = [: ]
+        mutableLinkAttributes[kCTForegroundColorAttributeName as String] = UIColor.textTint
+        mutableLinkAttributes[kCTUnderlineStyleAttributeName as String] = NSNumber(value: true)
+
+        var mutableActiveLinkAttributes: [AnyHashable: Any] = [:]
+        mutableActiveLinkAttributes[kCTForegroundColorAttributeName as String] = UIColor.brown
+        mutableActiveLinkAttributes[kCTUnderlineStyleAttributeName as String] = NSNumber(value: false)
+        if let label = label {
+            label.linkAttributes = mutableLinkAttributes
+            label.activeLinkAttributes = mutableActiveLinkAttributes
+        }
+    }
+
+    func addShadowToTitleLabel(for button: UIButton?) {
+        button?.titleLabel?.layer.shadowColor = UIColor.black.cgColor
+        button?.titleLabel?.layer.shadowOpacity = 0.3
+        button?.titleLabel?.layer.shadowRadius = 1
+        button?.titleLabel?.layer.shadowOffset = CGSize(width: 0.0, height: 2.0)
+    }
+
+    func addHorizontalLine(to view: UIView?, andHeight height: CGFloat) {
+        self.setMaxHeightIfGreaterFor(view, withHeight: height)
+        let offset = (view?.frame.size.height ?? 0.0) + 1
+        let lineView = UIView(frame: CGRect(x: (view?.frame.size.width ?? 0.0) / 15 - 10, y: offset, width: view?.frame.size.width ?? 0.0, height: 1))
+        lineView.backgroundColor = UIColor.utilityTint
+        view?.addSubview(lineView)
+    }
+
+    func getInformationTitleLabel(withTitle icon: UIImage?, atXPosition xPosition: CGFloat, atYPosition yPosition: CGFloat, andHeight height: CGFloat) -> UIImageView? {
```

Comment:
The parameter and return value should not be optional. Also, set the scope to fileprivate.